### PR TITLE
[Feature] Collect basic numa info from /sys by default without numad or numactl.

### PR DESCRIPTION
**Pull request by lherbolt**
Hi Bryn,
I know you collect the info from numad and numactl but in case you don't have it installed 
it's still good to have the basic info at least.

Lukas

### CODE REVIEW

**Comment by bmr-cymru** [comment]
Please fix your subject (there is no plugin or subsystem named 'Feature' ';-).

File: sos/plugins/kernel.py [hunk 1]
```diff
@@ -81,7 +82,10 @@ def setup(self):
             "/proc/timer*",
             "/proc/lock*",
             "/proc/misc",
-            "/var/log/dmesg"
+            "/var/log/dmesg",
+            numa_path + "node*/meminfo",
+            numa_path + "node*/cpulist",
+            numa_path + "node*/distance"
```
Comment (bmr-cymru) [inline sos/plugins/kernel.py]: Why would we put all this in the `kernel` plugin, when everything else numa-related is in `numa`? It seems unnecessarily confusing...

To put this into the `numa` plugin all you'd need to do would be to add a file trigger for a relevant path.

Generally we try to keep plugins small and specific today - `kernel` is one that has been around a very long time and is a "catch all" with no trigger conditions and a pretty weak organisation so we try not to add stuff there unless there is really nowhere else for it to go.

(I'd be very open, for e.g., to a patch moving all the remaining memory `/proc` files to `memory`, and splitting off driver, interrupt, and other bits to their own, dedicated plugins).

File: sos/plugins/kernel.py [hunk 2]
```diff
@@ -81,7 +82,10 @@ def setup(self):
             "/proc/timer*",
             "/proc/lock*",
             "/proc/misc",
-            "/var/log/dmesg"
+            "/var/log/dmesg",
+            numa_path + "node*/meminfo",
```
Comment (bmr-cymru) [inline sos/plugins/kernel.py]: Path concatenation should use `os.path.join`.

Comment (lherbolt) [inline sos/plugins/kernel.py]: Ok, I'll move it to numa and will try to fix the above issues.
Feel free to reject this request I'll create new one.

**Comment by pmoravec** [comment]
FYI no need to reject the PR, it is sufficient to amend/modify the commits (i.e. follow http://stackoverflow.com/questions/1186535/how-to-modify-a-specified-commit-in-git if uncertain on this) and force push to your local branch - the PR will be updated by the new commits properly.

File: sos/plugins/kernel.py (hunk 2 above)
Comment (bmr-cymru) [inline sos/plugins/kernel.py]: You can always just force-push an update to this branch - saves churning new PR#s.

**Comment by bmr-cymru** [comment]
:+1:

**Comment by bmr-cymru** [comment]
Ping.. any news on the updated patch?

**Comment by bmr-cymru** [comment]
We will be releasing 3.4 next month - I would like to include this change but there has been no update since last April.

The changes requested are minor so please do update the pull request if you'd like this to make the release.

**Comment by bmr-cymru** [comment]
There are now two unrelated commits on this branch (you seem to be doing all your work in your own `master` branch - it is better to use a single branch per topic to avoid this kind of conflict).

You are also working on a tree [from August 2015](https://github.com/lherbolt/sos) - please don't do that (every time I try to rebase your patch it conflicts with stuff from 2-300 commits ago!) - it makes it a real pain in the arse to deal with, considering this is just one small/simple patch.

For this one I'll straighten them out & get it merged as it's been sitting here for a long time but it's git "best practice" to keep things on their own branch, and rebased close to the current master.

**Comment by bmr-cymru** [comment]
I'll also assume a Signed-off-by from you for this change...

**Pull request closed without merging**